### ISSUE TITLE
Non-interactive mode for servers

### ISSUE THREAD

**Issue by Disinterpreter** [multitheftauto/mtasa-blue#1255]
**Is your feature request related to a problem? Please describe.**
We need a simple console programs for some modern solutions. For example manipulation with a docker or a systemd unit. I want to be able read the logs from `journalctl` or `docker log`

**Describe the solution you'd like**
I'd like to make a config option to the `mtaserver.conf` like `non-interactive`
after enabling, server will be show only stdout data.

**Additional context**
Thanks for reading!

**Comment by Disinterpreter** [comment]
Yes, after it i found the '-n' option (it solved my issue), but I want to set up it in config file.

**Comment by AlexTMjugador** [comment]
Out of curiosity, what's the advantage of setting the -n option via the configuration file instead on the command line parameters for your use case? AFAIK the MTA:SA server doesn't install any systemd or System-V service file, so you have to roll out your own, and you can make them so the server is launched with that parameter.

**Comment by Disinterpreter** [comment]
> 
> 
> Out of curiosity, what's the advantage of setting the -n option via the configuration file instead on the command line parameters for your use case? AFAIK the MTA:SA server doesn't install any systemd or System-V service file, so you have to roll out your own, and you can make them so the server is launched with that parameter.

There aren't advantages, just more convenient config in the mtaserver.conf

**Comment by qaisjp** [comment]
Sounds fine, assuming that config parsing is done early enough to make this possible
